lib/src/visitor/async_evaluate.dart

```diff
@@ -1580,6 +1581,7 @@ class _EvaluateVisitor
 
     _verifyArguments(positional.length, named, IfExpression.declaration, node);
 
+    // ignore: prefer_is_empty
```

Comment:
I think this qualifies as a bug for `prefer_is_empty`. Have you filed  one?

Reply:
It seems valid to me... `positional.isNotEmpty` would be be better style than `positional.length > 0` almost all the time, except for here where it's contrasting with other length checks.